corehq/apps/app_manager/suite_xml/post_process/endpoints.py

```diff
@@ -48,7 +48,8 @@ def update_suite(self):
                 for form in module.get_suite_forms():
                     if form.session_endpoint_id:
                         self.suite.endpoints.append(self._make_session_endpoint(
-                            form.session_endpoint_id, module, form))
+                            form.session_endpoint_id, module, form,
+                            respect_relevancy=getattr(form, 'respect_relevancy')))
```

Comment:
I don't understand why you use `getattr` here? The attribute name is not dynamic, nor is there a default value.

Reply:
it does not
```
>>> d = {}
>>> d.get("a")
>>> d.gettt("a")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict' object has no attribute 'gettt'
>>> getattr(d,'get')
<built-in method get of dict object at 0x1008fb480>
>>> getattr(d,'gettt')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict' object has no attribute 'gettt'
>>> 
```

Reply:
I was just using a dict because I knew it had a `get` function.

Reply:
I just meant that `getattr(form, 'respect_relevancy')))` has not advantage over `form.respect_relevancy` if you don't provide a default value or have a dynamic attribute name.